client.go

```diff
@@ -1547,6 +1559,53 @@ func (c *ToznySDKV3) WriteFile(ctx context.Context, recordType string, plain map
 	return fileRecord, nil
 }
 
+type ReadFileOptions struct {
+	RecordID         string
+	DownloadFileName string
+}
+
+// ReadFile downloads and decrypts the file from the record
+func (c *ToznySDKV3) ReadFile(ctx context.Context, options ReadFileOptions) error {
+	secretUUID, err := uuid.Parse(options.RecordID)
+	if err != nil {
+		return err
+	}
+	fileResp, err := c.E3dbPDSClient.GetFileRecord(ctx, secretUUID)
+	if err != nil {
+		return err
+	}
+	fileURL := fileResp.Metadata.FileMeta.FileURL
+	downloadRequest := file.DownloadRequest{
+		URL:               fileURL,
+		EncryptedFileName: DefaultDownloadedFileName,
+	}
+	downloadResponse, err := file.DownloadFile(downloadRequest)
+	if err != nil || downloadResponse != "" {
```

Comment:
Nope, the pattern here is
"I call a function, it returns two values, the second is an error value. If I want to know if it succeeded, I should only have to check the error value"

so if downlaodResponse being == "" is an error, file.DownloadFile should return a non-nil error and save callers from having to be super paranoid and check the error and other values.